### PR TITLE
Create 0x3D0AfE41c7806159Ac52E370C0Db2361826cc1f7

### PR DESCRIPTION
**Pull request by digitalgangster**

### CODE REVIEW

File: logos/0x3D0AfE41c7806159Ac52E370C0Db2361826cc1f7 [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (digitalgangster) [inline logos/0x3D0AfE41c7806159Ac52E370C0Db2361826cc1f7:1]: logo

**Review by digitalgangster — COMMENTED** [review]
logo

**Comment by Unsehrtain** [comment]
Duplicate, but this one is more correct than your previous. still requires fixing

(Folder name needs to be named with the token address)
-Please name it logo.png

**Comment by Unsehrtain** [comment]
closing due to no fix

**Pull request closed without merging**